src/main/java/teammates/common/datatransfer/FeedbackParticipantType.java

```diff
@@ -101,4 +127,18 @@ public String toSingularFormString() {
             return super.toString();
         }
     }
+
+    /**
+     * Getter method for JSTL to access the {@code displayNameGiver} property directly
```

Comment:
You don't need to mention "for JSTL". In fact, this can be as simple as "Gets the ABC property".